download_data.sh

```diff
@@ -3,7 +3,7 @@
 pushd /whosonfirst
 mkdir -p meta
 
-for layer in country disputed dependency macroregion region macrocounty county localadmin locality borough neighbourhood
+for layer in ocean marinearea continent empire country disputed dependency macroregion region macrocounty county localadmin locality borough neighbourhood
```

Comment:
For Mapzen Search we now use the whosonfirst importer download script. Can we include that repo here and use it instead? Keeping two download scripts up to date feels excessive, and this one isn't used (at least by us) anymore.